e2e-tests/native/test/pageobjects/landing.page.ts

```diff
@@ -0,0 +1,22 @@
+import {Page} from "./page";
+import {Selector} from "../Selector";
+
+class LandingPage extends Page{
+    constructor() {
+        super('Landing-Page');
+    }
+
+    get cities() {
+        return $$('~City-Entry')
+    }
+
+    get search() {
+        return $('~Search-Input')
+    }
+
+    async city(name: string) {
+        return $(new Selector().ByText(name).build())
```

Comment:
So we need the Selector class for text and normal strings for accessibility ids, right?

Reply:
that's correct. It is possible to implement selection also with the selector class. Then we could chain `ByText` and `ByAccessibilityId`. I did not implement it, as I thought that we would like to avoid platform specific selectors as much as possible. But it's probably interesting to think about this now to avoid later refactoring.

Reply:
I agree totally with you, if it works directly without custom selectors I don't see a reason why we should make it more complicated. But I am not really familiar with the topic...